XiEditor/Storage/SettingsProvider.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SettingsStore.swift
+//  XiEditor
+//
+//  Created by Alex Demishkevych on 10/13/18.
+//  Copyright © 2018 Raph Levien. All rights reserved.
+//
+
+import Foundation
+
+protocol SettingsProviding {
+    func fetchingRecentlyUsedLanguages() -> [String]
```

Comment:
This should probably be called `fetchRecentlyUsedLanguages` ("fetching" sounds very weird to me).

Reply:
@jansol yes, `fetchRecentlyUsedLanguages` definitely sounds better 👍🏻